lib/iam_ssoe_oauth/session_manager.rb

```diff
@@ -65,7 +67,7 @@ def build_identity(iam_profile)
 
     def build_session(access_token, user_identity)
       @session = IAMSession.new(token: access_token, uuid: user_identity.uuid)
-      @session.save
+      @session
```

Comment:
Idea is to not save `session` until *after* validating, since otherwise a subsequent request with the token would find the cached session.

Reply:
Like the change to saving after validating 👍 RubyMine is telling me you don't need line 70 as 69 also returns the session object, but maybe it's more clear to humans as is? 🤷‍♂️